tests/plugins/test_orchestrate_build.py

```diff
@@ -363,20 +380,24 @@ def test_orchestrate_build(tmpdir, caplog, config_kwargs,
         workflow.plugin_workspace[ReactorConfigPlugin.key][WORKSPACE_CONF_KEY] =\
             ReactorConfig(reactor_dict)
     else:
-        reactor_dict = {'version': 1, 'clusters': clusters}
+        reactor_dict = {
+            'version': 1,
+            'clusters': clusters,
+            'odcs': {'api_url': None, 'insecure': False},
+        }
```

Comment:
I think rather than adding odcs config here, it might be better to remove the odcs params from `expected_kwargs` to test what happens when odcs is not configured (which is a valid case)